### DISCUSSION TITLE
Is it possible to perform distributed load test using on-prem machines or google cloud instances (GCP)

### DISCUSSION THREAD

**Discussion by rrnareshkumar** [artilleryio/artillery#2096] [Artillery]
Hi ,

I noticed that it is possible to perform distributed load test  on AWS Lambda but may i know if it is possible to perform distributed load test using on-prem machines or google cloud instances (GCP) ?  Please suggest.

If it is possible , please share the reference docs/links, thanks.

**Comment by hassy** [comment]
hi @rrnareshkumar 👋 Not yet. We're focusing on building the best possible developer experience on AWS Lambda and AWS Fargate right now, though we will definitely look at adding support for GCP (as well as Azure) at some point.

**Reply by rrnareshkumar** [reply]
thanks for the update..

**Comment by TCRodriguez** [comment]
Hey there @hassy!

First off, just wanted to say Artillery is an incredible piece of software and its made quickly standing up some initial load testing runs on AWS Fargate a breeze.

That being said, my company is migrating over to GCP. I see that Azure support has since been added since the last activity in discussion. Was curious if there is movement at all towards implementing a GCP integration?

**Reply by hassy** [reply]
thanks @TCRodriguez! GCP support is coming (via Google Kubernetes Engine), some time in the next couple of weeks.

**Reply by TCRodriguez** [reply]
Hey @hassy !

That's exciting news. Any update on the status of this?

**Reply by TCRodriguez** [reply]
@hassy Hey there! Just pinging on this to see if there is any timeline update. My team is needing to figure out how to get load testing running on GCP and we'd like to still be able to use our scripts that we were using when we had access to AWS and AWS Fargate.

**Reply by joeklos** [reply]
Hey, @hassy and @TCRodriguez!

Just wanted to share some context from our side: We evaluated a ton of load testing tools and Artillery stood out as the best fit. We’ve loved using it so far! The only blocker for us now is GCP support, since our entire stack runs there.

Do you have any updates on the timeline for the integration? We’d need it in place by Q3 to fully adopt Artillery for our testing needs.

Thanks for all your work. Really hoping we can make this happen. 🙏

**Comment by TCRodriguez** [comment]
Hey @hassy! Hoping to get any update on the status of the GCP support you mentioned back in March? (https://github.com/artilleryio/artillery/discussions/2096#discussioncomment-12399344)

> thanks @TCRodriguez! GCP support is coming (via Google Kubernetes Engine), some time in the next couple of weeks.
